server/src/server.ts

```diff
@@ -34,10 +34,6 @@ connection.onInitialize((params): InitializeResult => {
   const workspaceRoot = params.rootPath ?? ''

```

Comment:
This commit is duplicated in https://github.com/savoirfairelinux/vscode-bitbake/pull/15/commits